bindings/python/iota_sdk/utils.py

```diff
@@ -128,13 +129,22 @@ def hash_transaction_essence(essence) -> HexStr:
         })
 
     @staticmethod
-    def verify_ed25519_signature(signature: HexStr, message: HexStr, address: Ed25519Address) -> bool:
-        """Verifies the Ed25519Signature for a message against an Ed25519Address.
+    def verify_ed25519_signature(self, signature: Ed25519Signature, message: HexStr):
+        """Verifies en Ed25519 Signature for a message.
         """
-        return _call_method('verifyEd25519Signature', {
+        return self._call_method('verifyEd25519Signature', {
+            'signature': signature,
+            'message': message,
+        })
+
+    @staticmethod
+    def verify_secp256k1_ecdsa_signature(self, public_key: HexStr, signature: HexStr, message: HexStr):
+        """Verifies an Evm Signature for a message.
+        """
+        return self._call_method('verifySecp256k1EcdsaSignature', {
```

Comment:
```suggestion
    def verify_secp256k1_ecdsa_signature(public_key: HexStr, signature: HexStr, message: HexStr):
        """Verifies an Evm Signature for a message.
        """
        return _call_method('verifySecp256k1EcdsaSignature', {
```
Same change above for verify_ed25519_signature

Reply:
Ah, right, thanks.